MAINTAINERS.md

```diff
@@ -20,6 +19,13 @@ describes governance guidelines and maintainer responsibilities.
 * James Peach ([jpeach](https://github.com/jpeach))
 * Daneyon Hansen ([danehans](https://github.com/danehans))
 
+## SIG Community
+
+| Owner | GitHub ID | Affiliation |
+| --------------- | --------- | ----------- |
+| Orlin Vasilev | [OrlinVasilev](https://github.com/OrlinVasilev) | [VMware](https://www.github.com/vmware/) |
+| Alex Xu | [xaleeks](https://github.com/xaleeks) | [VMware](https://www.github.com/vmware/) |
+
```

Comment:
Do you want to add SIG Docs here right now and put yourself as owner, and perhaps the technical working group as members?